com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/DeferredTile.shader

```diff
@@ -305,6 +305,7 @@ Shader "Hidden/HDRP/DeferredTile"
             #pragma multi_compile _ OUTPUT_SPLIT_LIGHTING
             #pragma multi_compile _ DEBUG_DISPLAY
             #pragma multi_compile _ SHADOWS_SHADOWMASK /// Variant with and without shadowmask
+            #pragma multi_compile SCREEN_SPACE_SHADOWS_OFF SCREEN_SPACE_SHADOWS_ON
```

Comment:
Shader

Reply:
For the stripper. If you don't give a name to the "implicit" version, i.e
#pragma _ SCREEN_SPACE_SHADOWS
 then you can't reject the implcit one. Here we want to reject one or the other, they are exclusive, 

this is different to have always the implicit and deciding to have the second or not like for debug
#pragma multi_compile _ DEBUG_DISPLAY